{{cookiecutter.project_slug}}/clients/web/vue3/package.json

```diff
@@ -3,53 +3,50 @@
   "version": "1.0.0",
   "description": "{{ cookiecutter.project_name }} Client Application",
   "private": true,
+  "type": "module",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "cypress": "source ../.env && npx cypress open",
-    "test:unit": "vue-cli-service test:unit",
-    "test:e2e": "vue-cli-service test:e2e",
-    "lint": "vue-cli-service lint",
-    "heroku-postbuild": "vue-cli-service build"
+    "serve": "vite",
+    "build": "vue-tsc && vite build",
+    "preview": "vite preview",
+    "test": "vitest run",
+    "test:dev": "vitest",
+    "cypress:dev": "cypress open",
+    "lint": "eslint --ext .js,.vue --ignore-path .gitignore --fix src",
+    "prettier": "prettier . --write",
+    "heroku-postbuild": "vue-tsc && vite build"
   },
   "dependencies": {
-    "@headlessui/vue": "^1.7.16",
-    "@heroicons/vue": "^2.0.18",
-    "@thinknimble/tn-forms": "^3.1.8",
-    "@thinknimble/tn-models": "^2.0.0",
+    "@headlessui/vue": "1.7.16",
+    "@heroicons/vue": "2.0.18",
+    "@thinknimble/tn-forms": "3.1.8",
+    "@thinknimble/tn-models": "2.2.0",
     "@thinknimble/tn-utils": "2.0.1",
-    "autoprefixer": "^10.4.15",
-    "axios": "^1.5.0",
-    "core-js": "^3.32.1",
-    "g": "^2.0.1",
-    "js-cookie": "^3.0.5",
-    "postcss": "^8.4.28",
-    "tailwindcss": "3.3.3",
+    "axios": "1.5.0",
+    "js-cookie": "3.0.5",
     "vue": "^3.3.4",
-    "vue-router": "^4.2.4",
-    "vuex": "^4.1.0",
-    "vuex-persistedstate": "^4.1.0",
-    "zod": "3.22.2"
+    "vue-router": "4.2.4",
+    "vuex": "4.1.0",
+    "vuex-persistedstate": "4.1.0",
+    "zod": "3.21.4"
```

Comment:
@lakardion I know this is a tiny change but how come we're going down a version here?
Similarly with vuex changing from ^4.1.0 to 4.1.0

Reply:
@mayakeeley